GoogleForCreators/web-stories-wp
# Dashboard: Display related templates on template details page

## Context

Show the _real_ related templates.

## Summary

Display templates that are related in the `Related Templates` section.

## Relevant Technical Choices

n/a

## To-do

n/a

## User-facing changes

Should see related templates instead of random ones!!

## Testing Instructions






This PR can be tested by following these steps:

1. Go to the `Explore Templates` tab in the dashboard.
2. Click `view details` on a template
3. Scroll to see related templates at the bottom of the page.
4. The displayed templates are all related by some metadata. Try other templates and see the list change.


## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #8827
